test/unit/core/type_traits/range_iterator_test.cpp

```diff
@@ -184,84 +194,89 @@ TEST(range_and_iterator, difference_type_)
+                                        size_t,
+                                        size_t>;
 
     expect_same_types<type_list_example, comp_list>();
 }
 
 TEST(range_and_iterator, innermost_value_type_)
 {
-    using type_list_example = type_list<typename innermost_value_type<std::vector<int>>::type,                    // long
-                                        innermost_value_type_t<std::vector<int>>,                                 // short
-                                        innermost_value_type_t<std::vector<std::vector<int>>>,                    // two-level
-                                        innermost_value_type_t<std::ranges::iterator_t<std::vector<int>>>,        // iterator
-                                        innermost_value_type_t<std::ranges::iterator_t<std::vector<int> const>>>; // const_iterator
-
-    using comp_list = type_list<int,
-                                int,
-                                int,
-                                int,
-                                int>;
+    using type_list_example = seqan3::type_list<typename seqan3::innermost_value_type<std::vector<int>>::type,
+                                                                                                       // long
+                                                seqan3::innermost_value_type_t<std::vector<int>>,      // short
+                                                seqan3::innermost_value_type_t<std::vector<std::vector<int>>>,
+                                                                                                       // two-level
+                                                seqan3::innermost_value_type_t<
+                                                    std::ranges::iterator_t<std::vector<int>>>,        // iterator
+                                                seqan3::innermost_value_type_t<
+                                                    std::ranges::iterator_t<std::vector<int> const>>>; // const_iterator
+
+    using comp_list = seqan3::type_list<int,
+                                        int,
+                                        int,
+                                        int,
+                                        int>;
 
     expect_same_types<type_list_example, comp_list>();
 }
 
 TEST(range_and_iterator, dimension)
 {
-    EXPECT_EQ(dimension_v<std::vector<int>>,                            1u);
-    EXPECT_EQ(dimension_v<std::ranges::iterator_t<std::vector<int>>>,                1u);
-    EXPECT_EQ(dimension_v<std::vector<std::vector<int>>>,               2u);
-    EXPECT_EQ(dimension_v<std::ranges::iterator_t<std::vector<std::vector<int>>>>,   2u);
+    EXPECT_EQ(seqan3::dimension_v<std::vector<int>>,                                        1u);
+    EXPECT_EQ(seqan3::dimension_v<std::ranges::iterator_t<std::vector<int>>>,               1u);
+    EXPECT_EQ(seqan3::dimension_v<std::vector<std::vector<int>>>,                           2u);
```

Comment:
don't align them, switch them if you think it's more readable

```suggestion
    EXPECT_EQ(2u, seqan3::dimension_v<std::vector<std::vector<int>>>);
```